src/widgets/aboutDialog.ui

```diff
@@ -21,6 +21,11 @@ Kukuh Syafaat
 Åke Engelbrektson</property>
     <property name="artists">Tobias Bernard
 J.P. MacDonald</property>
+    <property name="documenters">TheEvilSkeleton</property>
```

Comment:
@TheEvilSkeleton what name should I use here?

Reply:
TheEvilSkeleton should be fine. Also thank you :)